### DISCUSSION TITLE
C++ Accelerated

### DISCUSSION THREAD

**Discussion by Tlunch** [brandon-rhodes/python-sgp4#122] [General]
I am unsure if I missed something in the documentation. Is there any way of disabling the c++ accelerated feature programatically for testing purposes?

**Comment by brandon-rhodes** [comment]
I think you can simply import the Python class directly:

```
from sgp4.model import Satrec
```

See whether that gives you the un-accelerated class.

**Comment by Tlunch** [comment]
Ah I see rather than go through api. Thanks for the speedy response
